packages/web/src/fixtures/dev-kit/client.ts

```diff
@@ -358,3 +358,91 @@ export const waitForCreateMetrics = async (web3: Web3, propertyAddress: string):
       .catch(reject)
   })
 }
+
+export const detectStokens = async (web3: Web3, propertyAddress: string, accountAddress: string) => {
+  const client = newClient(web3)
+  if (client) {
+    return
+  }
+  const TokenIdList = await devClient.createDetectSTokens(
+    // @k3nt0w: ignore compile error until implement get stoken contract address
+    // @ts-ignore
+    await getContractAddress(client, 'sTokens')
+  )(propertyAddress, accountAddress)
+  return TokenIdList
+}
+
+export const getStokenPositions = async (web3: Web3, propertyAddress: string, sTokenID: number) => {
+  const client = newClient(web3)
+  if (client) {
+    return client.sTokens(propertyAddress).positions(sTokenID)
```

Comment:
```ts
return client?.sTokens().....
```
How about this?

Reply:
`client.sTokens` takes `addresses.eth.main.sTokens` as the argument, not a property address. Since `addresses.eth.main.sTokens` doesn't exist yet, I've published an improved version that returns 0x0 address.

```ts
import {addresses} from '@devprotocol/dev-kit'

console.log(addresses.eth.main.sTokens)
```

Please use **@devprotocol/dev-kit@4.6.1-alpha.0**